resources/migrations/0098_add_panoroma_image_type.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.11 on 2022-06-08 08:31
```

Comment:
Recreate this when you have updated the string.

Reply:
Updated the migrations file.